packages/frontend/core/src/components/page-list/operation-cell.tsx

```diff
@@ -113,13 +118,26 @@ export const PageOperationCell = ({
     );
   }, [page.id, favAdapter, t]);
 
+  const onToggleFavoritePageOption = useCallback(() => {
+    track.allDocs.list.docMenu.toggleFavorite();
+
+    onToggleFavoritePage();
+  }, [onToggleFavoritePage]);
+
   const onDuplicate = useCallback(() => {
     duplicate(page.id, false);
     track.allDocs.list.docMenu.createDoc({
       control: 'duplicate',
     });
   }, [duplicate, page.id]);
 
+  const handleRemoveFromAllowList = useCallback(() => {
+    if (onRemoveFromAllowList) {
+      onRemoveFromAllowList();
+      track.collection.docList.docMenu.removeSpecialFilter();
```

Comment:
deleteOrganizeItem({ type: 'doc' })

Reply:
But it is not `organizeItem`, this function is to remove doc from collection.